src/index.tsx

```diff
@@ -638,72 +638,72 @@ export class App extends React.Component<{}, AppState> {
 
                   let deltaX = 0;
                   let deltaY = 0;
-                  selectedElements.forEach(element => {
```

Comment:
The `forEach` wasn't needed since before we check if `selectedElements.length === 1`

Reply:
I recently noticed the diamond flip bg bug too. But since you're only normalizing on mouseup, the bg isn't fix during the resize itself. Is it a problem to normalize during move?

Reply:
Since `mousemove` fires too often I didn't want to add extra code there to avoid hindering performance. However, it would be nice to normalize during move. I'll give it a try.